runtime/src/iree/tools/utils/vm_util.h

```diff
@@ -52,10 +52,6 @@ inline Status PrintVariantList(iree_vm_list_t* variant_list,
   return PrintVariantList(variant_list, max_element_count, &std::cout);
 }
 
-// Creates the default device for |driver| in |out_device|.
-// The returned |out_device| must be released by the caller.
-Status CreateDevice(const char* driver_name, iree_hal_device_t** out_device);
```

Comment:
Nice, less C++ code in vm_util :D

(see also the comment at the top of this file)
```
// NOTE: this file is not best-practice and needs to be rewritten; consider this
// appropriate only for test code.
```

Reply:
slowly chipping away at it :)